### ISSUE TITLE
Bluetooth service not enabled by default after gnome installation

### ISSUE THREAD

**Issue by azomDev** [archlinux/archinstall#2678]
### Which ISO version are you using?

2024-08-01 (but it happened on others I tested in the past)

### The installation log

[install.log](https://github.com/user-attachments/files/16947044/install.log)

#### Description of the issue

When using the archinstall script to install gnome, I need to manually run `sudo systemctl enable bluetooth` to be able to use bluetooth. If I don't do this, trying to toggle bluetooth with gnome quick settings or in the settings app does not work (the toggle does not switch on and it does not try to scan/list/connect bluetooth devices).

I don't know if this is intended or if this is even a problem with archinstall, but given that this problem appears right when using gnome after the install, I felt it was accurate to create the issue here.

#### More info

I get this when running `systemctl status bluetooth`:
```
○ bluetooth.service - Bluetooth service
     Loaded: loaded (/usr/lib/systemd/system/bluetooth.service; disabled; preset: disabled)
     Active: inactive (dead)
       Docs: man:bluetoothd(8)
```

`systemctl restart bluetooth` also works as a temporary fix, but has to be runned each time after a reboot.

**Comment by JTEpic** [comment]
It does the same in KDE because of the service not being enabled, however past issues have said this is more of a post-install thing, although it would be nice if it was enabled by default for specific environments. Better yet a list of what archinstall does not do and how to do it would be nice if others would be in favor of this.

**Comment by azomDev** [comment]
I think it would be beneficial to enable Bluetooth in certain environments. GNOME, for instance, offers a seamless "out of the box" experience, and having Bluetooth enabled by default would fit well since it doesn’t require additional installations. I can't speak for other desktop environments, but KDE also comes to mind in this context.

> Better yet a list of what archinstall does not do and how to do it would be nice if others would be in favor of this.

I agree with this approach. While debugging my issue wasn’t particularly difficult, it did take some time because I initially thought it was a GNOME-specific problem. I still think it would be an improvement to have it enabled by default in environments where there's a built-in setting for Bluetooth such as GNOME or KDE.

**Comment by Torxed** [comment]
It's totally fine to create a issue for these kinds of things and discussions are encouraged. However this is intentional. The main reason why this isn't done is as @JTEpic mentions because it's an easy post-installation step, but also by default we don't want to *"force"* things on any user. This is especially true when it enables connectivity/means of accessing the machine.

Because we currently have no *"post-installation"* submenu/system/concept there's also no place to add this currently. Personally I also don't think we want to add more menu options as it's already a bit cluttered.

And this "default" in particular is one that is going to be a divider among users. So we need an option somewhere. Some *"just want stuff working"* and some don't want anything happening without them enabling it. But I get how this could be useful from a desktop perspective, especially when the desktop doesn't give any indications of what's going "wrong" *(I'm looking at you Gnome)*.

I definitely think a post-install menu could be added, maybe to add/change to the current behavior of asking if the user wants to drop into a shell in the new system where there are some optimizations/easy changes to be added.

**Comment by mrust1219** [comment]
Even if you install Arch the manual way this is the case. Flagship distros like Fedora do this to be ready to use out of the box. Arch is a minimal DIY distro. A simple "sudo systemctl enable bluetooth" will resolve this issue.

**Comment by azomDev** [comment]
Fair enough. From my perspective, even on Arch Linux, users of GNOME probably aren’t looking for a super minimal setup, given that they’ve chosen GNOME. But I’m only speaking from my own experience here, so probably not a good representation of the users.

It's a small thing that I *feel* should be on by default for GNOME specifically since it's not really a minimal DE. Personally, it doesn’t bother me too much, and if including those features goes against the principles of Archinstall, I’m fine with that.
